tests/test_bigbio.py

```diff
@@ -0,0 +1,660 @@
+"""
+Unit-tests to ensure tasks adhere to QA for big-bio schema.
+"""
+import sys
+import datasets
+from datasets import load_dataset, Features
+from schemas import (
+    kb_features,
+    qa_features,
+    entailment_features,
+    text2text_features,
+    text_features,
+    pairs_features,
+)
+
+import unittest
+import importlib
+import argparse
+from collections import defaultdict
+from difflib import ndiff
+from pathlib import Path
+from pprint import pformat
+from typing import Iterator, Optional, Union, Iterable, List
+
+sys.path.append(str(Path(__file__).parent.parent))
+
+import logging as log
+
+log.basicConfig(level=log.INFO)
+
+
+_TASK_MAP = {
+    "kb": "kb",
+    "ner": "kb",
+    "ned": "kb",
+    "re": "kb",
+    "entities": "kb",
+    "relations": "kb",
+    "events": "kb",
+    "coref": "kb",
+    "qa": "qa",
+    "question-answering": "qa",
+    "entailment": "entailment",
+    "paraphrasing": "text_to_text",
+    "summarization": "text_to_text",
+    "translation": "text_to_text",
+    "text_to_text": "text_to_text",
+    "classification": "text",
+    "text": "text",
+    "pairs": "pairs",
+}
+
+_SCHEMA_MAP = {
+    "kb": kb_features,
+    "qa": qa_features,
+    "entailment": entailment_features,
+    "text_to_text": text2text_features,
+    "text": text_features,
+    "pairs": pairs_features,
+}
```

Comment:
following the example above, would it make sense to call this one `_SCHEMA_TO_FEATURES` ?